source/includes/app-services-cli-overview.rst

```diff
@@ -158,3 +158,69 @@ The CLI stores identifiers and configuration metadata in the
    * - | ``client_app_id``
        | ``string``
      - The human-readable Client App ID.
+
+.. _cli-profiles:
+
+CLI Profiles
+------------
+
+The CLI stores information about its users in a profile. This lets it
+remember information between commands. For example, when you log in with
```

Comment:
Nit: I think it's a general best practice to avoid anthropomorphizing in tech writing. Maybe we could reword this?  
```suggestion
The CLI stores information about its users in a profile. This lets you
run commands in a given context. For example, when you log in with
```